mache/machines/polaris.cfg

```diff
@@ -0,0 +1,47 @@
+# Options related to deploying an e3sm-unified conda environment on supported
+# machines
+[e3sm_unified]
+
+# the unix group for permissions for the e3sm-unified conda environment
+group = E3SM
+
+# the path to the directory where activation scripts, the base environment, and
+# system libraries will be deployed
+base_path = /eagle/E3SM/software/e3sm-unified
+
+
+# config options related to data needed by diagnostics software such as
+# e3sm_diags and MPAS-Analysis
+[diagnostics]
+
+# The base path to the diagnostics directory
+base_path = /eagle/E3SM/diagnostics
+
+# the unix group for permissions for diagnostics
+group = E3SM
+
+
+# The parallel section describes options related to running jobs in parallel
+[parallel]
+
+# parallel system of execution
+system = pbs
```

Comment:
Currently, PBS isn't supported by any of the downstream tools (notably Compass and Polaris) and we don't have a computing allocation on Polaris anyway.